src/app/features/tech-record/components/test-record-summary/test-record-summary.component.ts

```diff
@@ -10,6 +10,9 @@ import { VehicleTechRecordModel } from '@models/vehicle-tech-record.model';
 export class TestRecordSummaryComponent {
   @Input() testRecords: TestResultModel[] = [];
   @Input() vehicleTechRecord?: VehicleTechRecordModel;
+  @Input() currentPage: number = 1;
+
+  itemsPerPage = 5;
```

Comment:
I guess there is an argument that could be made that passing in a value for `itemsPerPage` is a good idea, however, seing as 5 items per page is the default in the pagination component, it's not strictly necessary to pass it down. You could argue there is no sense in having default values if you're never going to use them.

Reply:
`true`

Reply:
Moved most logic to component so the host only cares about rendering the rows.